src/dotnet/Chat.UI.Blazor/Components/ChatView/Items/ChatMessageMarkup.razor

```diff
@@ -20,6 +26,9 @@
     [Inject] private IActivePlaybackInfo ActivePlaybackInfo { get; init; } = null!;
     private ChatEntry Entry => Model.Entry;
     private Markup? Markup { get; set; } = null!;
+    private bool HasTextOnly => Markup?.Parts.All(x => x is PlainTextPart) ?? false;
```

Comment:
I guess PlainTextPart is a bit of a legacy in terms of design. Most likely we'll simply have to move TextToTimeMap to PlainTextPart & make it optional there, or even better, have something like `IPlayablePart`, which optionally has `TextToTimeMap` & time range. When they're there, the part is playable & so it might be rendered as words.